cross/.gitmirror

```diff
@@ -0,0 +1 @@
+Only contents of this folder, excluding subfolders, will be mirrored by the Git-TFS Mirror. 
```

Comment:
That should probably be a .gitmirrorall actually, as I also mirrored the other directories, I used a regular gitmirror as a rootfs directory may be generated there but it should never be committed, it's in .gitignore. Other Unix only components like the PAL are also under .gitmirror .